### PR TITLE
Tests: Apertus integration tests

### PR DESCRIPTION
**Pull request by andresnowak**
# What does this PR do?

This PR adds integration tests for the Apertus model, the tests are:
- Check short greedy generation
- Check short instruct model greedy generation
- Check model logits in bfloat16, and compare with XIELU cuda output
- Check model logits in float32, and compare with XIELU cuda output (I think float32 tests are not desired?)

Note: Right now I do an environment variable just to show the difference between experimental XIELU cuda and no XIELU cuda outputs (but if this isn't desirable I'll remove it and just do a test for one of the two versions instead)

Related PR: #39381 

@ArthurZucker

### CODE REVIEW

**Review by ArthurZucker — APPROVED** [review]
Thanks!

**Comment by ArthurZucker** [comment]
run-slow: apertus
